src/main/java/com/github/seratch/jslack/api/scim/ScimClient.java

```diff
@@ -0,0 +1,13 @@
+package com.github.seratch.jslack.api.scim;
+
+import com.github.seratch.jslack.api.methods.SlackApiException;
+import com.github.seratch.jslack.api.scim.request.UsersDeleteRequest;
+import com.github.seratch.jslack.api.scim.response.UsersDeleteResponse;
+
+import java.io.IOException;
+
+public interface ScimClient {
```

Comment:
Let's have a comment about the interface like https://github.com/davidkarlsen/jslack/blob/6270343fdaef54d2a679ffad2ead64ecd580cf0b/src/main/java/com/github/seratch/jslack/api/methods/MethodsClient.java#L100-L103